### PR TITLE
Tabs-Components

### PR DESCRIPTION
**Pull request by AnniqueN**

### CODE REVIEW

File: components/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -2,24 +2,25 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element;
```
Comment (Mykolb) [inline components/Dropdown/Dropdown.js:5]: This looks good!

File: index.html [hunk 2]
```diff
@@ -1,80 +1,100 @@
...
+				<a class="dropdown-link" href="https://lambdaschool.com/" target="_blank">Lambda</a>
+				<a class="dropdown-link" href="https://www.google.com/" target="_blank">Google</a>
+				<a class="dropdown-link" href="https://developer.mozilla.org/en-US/" target="_blank">MDN</a>
 			</div>
-			<img src="./lambda_logo_light_small.png" class="header" />
 		</div>
+		<img src="./lambda_logo_light_small.png" class="header" />
+	</div>
 
-		<div class="section">
```
Comment (Mykolb) [inline index.html:23]: Are you creating a carousel?

File: components/Tabs/Tabs.js [hunk 3]
```diff
@@ -3,49 +3,56 @@ class TabLink {
...
 
   select() {
     // Get all of the elements with the tabs-link class
     // const links;
-
+    const links = document.querySelectorAll(".tabs-link");
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
     // Array.from(links).forEach();
-
+    links.forEach(tab => tab.classList.remove('tabs-link-selected'))
```
Comment (Mykolb) [inline components/Tabs/Tabs.js:30]: You could use Array.from if you wanted covert the nodelist to an array. Ex:

```Array.from(links).forEach(tab => tab.classList.remove('tabs-link-selected'));```

**Review by Mykolb — COMMENTED** [review]
It looks good! MVP met, merging PR

File: index.html (hunk 2 above)
Comment (AnniqueN) [reply index.html:23]: I was going too but then i quit lol

**Pull request merged**